Tests/Marketplace/marketplace_services.py

```diff
@@ -737,17 +737,25 @@ def sign_pack(self, signature_string=None):
         finally:
             return task_status
 
-    def encrypt_pack(self, zip_pack_path, pack_name, encryption_key, extract_destination_path):
+    @staticmethod
+    def encrypt_pack(zip_pack_path, pack_name, encryption_key, extract_destination_path):
         try:
+            current_working_dir = os.getcwd()
             shutil.copy('./encryptor', os.path.join(extract_destination_path, 'encryptor'))
             os.chmod(os.path.join(extract_destination_path, 'encryptor'), stat.S_IXOTH)
-            current_working_dir = os.getcwd()
             os.chdir(extract_destination_path)
             output_file = zip_pack_path.replace("_not_encrypted.zip", ".zip")
             subprocess.call('chmod +x ./encryptor', shell=True)
             full_command = f'./encryptor ./{pack_name}_not_encrypted.zip {output_file} "' \
                            f'{encryption_key}"'
+
             subprocess.call(full_command, shell=True)
+            new_artefacts = os.path.join(current_working_dir, 'private_artifacts')
```

Comment:
I would recommend avoid hard coded values, and pass the output files as part of the script args.
In this case I think it is better to pass the path to save the encrypted file and non encrypted file.
But isn't the non encrypted file existed before?

Reply:
The path is relative to the existing path, and I feel this script already has enough arguments as it is, and there would be little added value here - this folder is only used for the artifacts anyway.
How strongly do you believe this change is necessary?

Reply:
If you decide to use a hard-coded value, I suggest maintaining it in a class/global-variable. This can help us maintain the use of this constant more easily. See `BucketUploadFlow` class in the same file for reference. What do you think?